### DISCUSSION TITLE
Copy current file name with path

### DISCUSSION THREAD

**Discussion by yusuphgammut** [helix-editor/helix#6957] [Q&A] [answered]
Hi people.

I was wondering if there's a way to copy into the clipboard the current file name with path. It can be full path, or just relative to the current working directory. I was thinking on using `:sh` with `pbcopy` or something like that, but I don't know if expansions are supported within typable commands.

**Comment by yusuphgammut** [comment]
Seems that there isn't a way for achieving that. I'll try to create a PR.

**Reply by kirawi** [reply, marked as answer]
You can try to pick up https://github.com/helix-editor/helix/pull/3393 if you're interested.

**Reply by yusuphgammut** [reply]
Ohh thanks, didn't knew about that PR. I'll check it!

**Comment by logankaser** [comment]
```
[keys.normal]
C-n = ':! echo -e "\e]52;;$(echo %{buffer_name} | base64)\007" > /dev/tty'
```
Binds copy file name to clipboard to `Ctrl+n`
Requires `echo` and `base64`, assumes linux

**Comment by tshepang** [comment]
is there a way to do this without using `:sh`
